utils/sentry_client.py

```diff
@@ -0,0 +1,155 @@
+import requests
+import json
+
+class SentryClient:
+  ORGANIZATION = "sentry"
+
+  def __init__(self, host, token):
+    self.host = host
+    self.auth_token = token
+
+  def do_sentry_api_call(self, method, path, slugs, payload=None):
```

Comment:
I like to prepend private methods with `_`

Reply:
Yep.  Client still needs a bit of work for better error handing and style changes like this.  Done.